conans/cli/commands/config.py

```diff
@@ -46,3 +49,17 @@ def config_home(conan_api, parser, subparser, *args):
     home = conan_api.config.home()
     ConanOutput().info(f"Current Conan home: {home}")
     return home
+
+
+@conan_subcommand(formatters={"json": lambda x: json.dumps(x, indent=4)})
```

Comment:
Maybe do like other commands and use `json_formatter` imported from `conans.cli.commands`?